### DISCUSSION TITLE
Weekly Governance Updates (26 April 2024)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#999] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0063 - Switching to new Drand mainnet network [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0063.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/567)
- FIP0065 - Ignore built-in market locked balance in circulating supply calculation [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0065.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/719)
- FIP0067 - PoRep Security Policy & Replacement Sealing Enforcement [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0067.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/727)
- FIP0074 - Remove cron-based automatic deal settlement [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0074.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/800)
- FIP0076 - Direct Data Onboarding [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0076.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/730)
- FIP0079 - Add BLS Aggregate Signatures to FVM [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0079.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/840)
- FIP0082 - Add support for aggregated replica update proofs [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0082.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/752)
- FIP0083 - Add Built-in Actor Events [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0083.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/754)
- FIP0084 - Remove Storage Miner Actor Method `ProveCommitSectors` [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0084.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/899)
- FIP0085: Convert f090 Mining Reserve Actor to Keyless Account Actor [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0085.md) [(Discussion)](https://github.com/filecoin-project/FIPs/discussions/901)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

* Update FIP0001 [(PR)](https://github.com/filecoin-project/FIPs/pull/850)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/799)
* On-chain Retrieval Expectations [(PR)](https://github.com/filecoin-project/FIPs/pull/862)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/861) 
* Introducing SealerID [(PR)](https://github.com/filecoin-project/FIPs/pull/993)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/890)

**Last Call** - *Final opportunity for community feedback!* 

- FIP0090: Non-Interactive PoRep [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0090.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/854). Last Call until 6 May 2024. 

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas model adjustment for user programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
- FIP0077 - Add Cost Opportunity For New Miner Creation [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0077.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/780)
- FIP0078 - Remove Restrictions on the Minting of Datacap [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0078.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/844)
- FIP0080 - Phasing Out Fil+ and Restoring Deal Quality Multiplier to 1x [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0080.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/774)
- FIP0081 - Introduce lower bound for sector initial pledge [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0081.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/847)
- FIP0086 - Fast Finality in Filecoin (F3) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/809)

 **:fire: HOT :fire: Discussion Topics**

- There is a proposal to enable fast retrieval on Filecoin by creating a second type of proof dubbed "Fast PoRep". Read more [here](https://github.com/filecoin-project/FIPs/discussions/988)
- Filecoin Foundation has recently released the FIP draft for a proposed update to FIP0001. This new proposal contains significant improvements to the existing governance models of Filecoin. Read the full draft [here](https://github.com/filecoin-project/FIPs/pull/850) 
- There is an ongoing discussion to make more explicit the roles and expectations of FIP Editors to improve the experience of the Filecoin community with the FIPs process. Read the full discussion and contribute [here](https://github.com/filecoin-project/FIPs/discussions/960)

---
**Other Governance Updates**

* Nv 22 Dragon upgrade successfully shipped on mainnet April 24 2024. Plans have kickstarted for nv23 and more information will be made available soon. To read more about nv23 upgrade check [here](https://github.com/filecoin-project/core-devs/discussions/149)
* Monday 29 April is our next Community Governance Call. Make sure you have our GCal link [here](https://calendar.google.com/calendar/embed?src=c_909343f97c15e8f23dda6e2612e62fcdee14bceabd8869abe4a52d793bf42b98%40group.calendar.google.com&ctz=Europe%2FLondon) to get the full information.

**Comment by 15012700225** [comment]
FIP0080  +1
